Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -147,13 +165,25 @@ private void InteractionManager_SourceLost(InteractionSourceState state)
 
         private void GestureRecognizer_TappedEvent(InteractionSourceKind source, int tapCount, Ray headRay)
         {
-            OnTap();
+            ProcessTap();
```

Comment:
Since we are touching this, I feel we should not get rid of the event args if people want to key off of number of taps etc.